src/v2/view-builder/utils/TimeZone.js

```diff
@@ -0,0 +1,35 @@
+export default {
```

Comment:
While these values are mapped to [IANA time zone identifiers](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), because they're surfaced as an enduser facing message, we'll want to make sure they are translated.

FYI: @nikhilvenkatraman-okta

Reply:
@nikhilvenkatraman-okta @jmelberg-okta Do we require all these time zones to be translated? That seems like a lot of work. Can we tackle that as a separate follow-on PR?

Reply:
I think the ask is to just move the string values into the login.properties file here so that it can get picked up for translation at the end of the month.

Reply:
@jmelberg-okta @nikhilvenkatraman-okta @huanliu-okta  These are standard values that can be pulled from the [CLDR](https://cldr.unicode.org/). Do we still want to keep this surfaced as end-user strings? Or do we want to take a look at an option to pull directly from CLDR programmatically.  Let me know if you have any questions or if this helps.

Reply:
`# {0} is the different timezone value`
`timezone.option.label = {0}`
can I just add this variable in the properties file?

Reply:
@nikhilvenkatraman-okta @evelynteo-okta  i have added the dropdown strings to the login.property.

Reply:
Discussed offline. Adding these values as arguments to the login.properties does not add any value, as these strings wont get translated in a different language. 

As mentioned, the right fix needs more investigation and will likely need us to pull these values from an external source. That can be done as a different JIRA.

Please revert the last commit and should be good to go 🚀